sass/tji/_charts.scss

```diff
@@ -1,6 +1,27 @@
 /*--------------------------------------------------------------
 ## CHARTS
 --------------------------------------------------------------*/
+
+.tji-chart {	
+	border: $tji-border;
+	padding: $tji-spacing-md $tji-spacing-sm;
+	margin: $tji-spacing-xs;
```

Comment:
since flexbox-grid classes (col-xs, col-md, etc...) already applies margin styles we shouldn't override these or it will break the grid layout flexbox-grid provides for us. Can we remove this margin style?

You'll see the chart is not centered because of this extra margin that's applied which override flexbox-grid's margin in the screen recording below.
![flexboxgrid](https://user-images.githubusercontent.com/3039125/43478672-1eff7196-94c4-11e8-8190-66b2eeade139.gif)

Reply:
I don't see any margin styles in flexbox grid. I think this problem is actually due to in unnecessary margin-right: 2em in _general.scss. I removed that and changed the chart style to only apply the margin on medium+ viewports. This looks good on my end, what do you think?